src/DotNetWorker/Invocation/DefaultFunctionExecutor.cs

```diff
@@ -1,20 +1,93 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.Azure.Functions.Worker.Pipeline;
 
 namespace Microsoft.Azure.Functions.Worker.Invocation
 {
     internal class DefaultFunctionExecutor : IFunctionExecutor
     {
+        private const string FunctionReturnType = "$WorkerReturn";
```

Comment:
A reserved binding name, to indicate that this comes via the return type directly.